internal/api/external.go

```diff
@@ -514,7 +514,7 @@ func (a *API) Provider(ctx context.Context, name string, scopes string) (provide
 
 	switch name {
 	case "apple":
-		return provider.NewAppleProvider(config.External.Apple)
+		return provider.NewAppleProvider(ctx, config.External.Apple)
```

Comment:
Unrelated to PR but is there any reason why we don't allow users to pass in scopes for Apple?

Reply:
apple only has 2 scopes and we're passing both of them in by default already